src/ReactiveUI/CommandBinding.cs

```diff
@@ -30,7 +33,7 @@ static CommandBinder()
         /// <param name="view">The View</param>
         /// <param name="viewModel">The View model</param>
         /// <param name="controlName">The name of the control on the view</param>
```

Comment:
re: seperate commit for controlProperty/controlName 👍 

re: extension method styling for xmldoc

@kentcb and @paulcbetts any thoughts/desires on this one?